tests/plugins/wysiwygarea/detached.js

```diff
@@ -0,0 +1,260 @@
+/* bender-tags: editor, 4462 */
+/* bender-ckeditor-plugins: basicstyles,toolbar */
+
+( function() {
+	'use strict';
+
+	function isSupportedEnvironment() {
+		// Ignore all IE versions below 11.
+		return !( CKEDITOR.env.ie && !CKEDITOR.env.edge && CKEDITOR.env.version < 11 );
+	}
+
+	bender.test( {
+		'test reattached editor contains the same data with observed default dom object': function() {
+			if ( !isSupportedEnvironment() ) {
+				assert.ignore();
+			}
+
+			var startupData = '<p>CKEditor4</p>';
+
+			bender.editorBot.create( {
+				startupData: startupData,
+				name: 'editor' + new Date().getTime()
+			}, function( bot ) {
+				var editorContainer = bot.editor.container,
+					editorContainerParent = editorContainer.getParent();
+
+				editorContainer.remove();
+				editorContainerParent.append( editorContainer );
+
+				CKEDITOR.tools.setTimeout( function() {
+					resume( function() {
+						var iframeElement = bot.editor.ui.space( 'contents' ).findOne( 'iframe' ),
+							editorData = iframeElement.getFrameDocument().getBody().getHtml();
```

Comment:
Can't we just use `bot.editor.getData()`?

Reply:
Sadly - we cannot :cry: It involves internal logic and it returns 'valid' data even if the iframe itself reloads and has no content. So with `bot.editor.getData()` tests could pass or fail due to wrong reasons.

For example `test reattached editor contains the same data with observed default dom object` can pass even without the fix. OTOH test that expects empty editor fails, because of `getData()`.